.github/workflows/label-base-token.yml

```diff
@@ -0,0 +1,41 @@
+name: Label PR if token is for base-goerli
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Get changed .json files
+        id: changed-files
+        uses: tj-actions/changed-files@v36
+        with:
+          files: '**.json'
+          separator: ','
+
+      - name: Check for 'base-goerli' is in changed files
+        id: check
+        run: |
+          FILES='${{ steps.changed-files.outputs.all_modified_files }}'
```

Comment:
is there a way to make this so it only looks at modified files under the `data/` directory?

Reply:
Will update so it only looks in data/ directory!